FF1Lib/Fun.cs

```diff
@@ -415,9 +444,9 @@ public void EnableModernBattlefield()
 			// Update Several 16 Byte LUTS related to cursors
 			for (int i = 0; i < 8; ++i)
 			{
-				Data[0x31F85 + i * 2] = (byte)(xCoord - 0x10);            // X Coord of Character targeting cursor
-				Data[0x31F86 + i * 2] = (byte)(yCoord + 4 + ((i % 4) * 0x1C));  // Y Coord of Character targeting cursor
-				Data[0x31F76 + i * 2] = (byte)(0xA7 + (Math.Min(i, 4) % 4 * 0x10)); // Y Coord of Command Menu cursor (last four are identical)
+				Data[0x31F85 + (i * 2)] = (byte)(xCoord - 0x10);            // X Coord of Character targeting cursor

```

Comment:
we don't have to do this, do we?

Reply:
this does not change the functionality, it is the result of choosing IDE0048 over IDE0047
https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0047-ide0048

basically, the formatter will automatically add parentheses anywhere that might cause confusion, so people later on dont make a mistake when reading the code.

Reply:
but that operator precedence you usually learn around the age of 10